### PR TITLE
Chore: Fixed broken stories and added more

### PR DESCRIPTION
**Pull request by indraraj**
- Added the helper text for the "Duplicate connector instance" button in the drawer
- Fixed Drawer story 
- Refactored ConnectorDrawer and ConnectorInfoTextList Component

Screenshot
image

### CODE REVIEW

File: src/app/components/ConnectorInfoTextList/ConnectorInfoTextList.tsx [hunk 1]
```diff
@@ -12,24 +14,25 @@ import {
...
 
-type AlertType = 'info' | 'warning' | 'danger' | undefined;
-
 export type ConnectorInfoTextListProps = {
   name: string;
   id: string;
   type?: string;
   bootstrapServer: string;
-  kafkaId: string | KafkaInstance | ReactNode;
+  KIData: string | KafkaInstance | ReactNode;
```
Comment (gashcrumb) [inline src/app/components/ConnectorInfoTextList/ConnectorInfoTextList.tsx:33]: Maybe `kafkaInstanceData`?

File: src/app/components/ConnectorDrawer/ConnectorDrawer.tsx [hunk 2]
```diff
@@ -72,21 +60,12 @@ export const ConnectorDrawer: FunctionComponent<ConnectorDrawerProps> = ({
...
-              kafkaId={connector.kafka.id}
-              owner={connector.owner!}
-              namespaceId={connector.namespace_id!}
-              createdAt={new Date(connector.created_at!)}
-              modifiedAt={new Date(connector.modified_at!)}
-              status={connector.status?.state!}
-              error={connector.status?.error}
+            <ConnectorDrawerComponent
+              currentConnectorRef={currentConnectorRef}
+              connectorData={connector}
```
Comment (gashcrumb) [inline src/app/components/ConnectorDrawer/ConnectorDrawer.tsx:65]: I think just `connector` would be fine for the property name.  But then wouldn't just the ref from the state machine be enough to pass through this component?  Passing both of these objects strikes me as redundant, it'd probably be better to keep it simpler by passing only the connector instance or the state machine reference.

File: src/app/components/ConnectorDrawer/ConnectorDrawer.tsx [hunk 3]
```diff
@@ -183,24 +142,17 @@ export const ConnectorDrawerPanelContent: FunctionComponent<ConnectorDrawerPanel
...
       })(getKIData, onKIError);
       // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [kafkaId]);
-
-    const selectActiveKey = (_: MouseEvent, eventKey: string | number) => {
-      setActiveTabKey(eventKey);
-    };
+    }, [connectorData]);
 
     const { connector } = useConnector(
```
Comment (gashcrumb) [inline src/app/components/ConnectorDrawer/ConnectorDrawer.tsx:157]: What's this connector vs `connectorData` that's being passed in?

File: src/app/components/ConnectorDrawer/ConnectorDrawer.tsx [hunk 4]
```diff
@@ -212,61 +164,89 @@ export const ConnectorDrawerPanelContent: FunctionComponent<ConnectorDrawerPanel
...
+  status: string;
+  error?: string;
+  connectorStatus: React.ReactNode;
+  connectorActions: React.ReactNode;
+  onClose: () => void;
+  onConnectorDetail: (id: string, goToConnectorDetails: string) => void;
+  onDuplicateConnector: (id: string) => void;
+};
+
+export const ConnectorDrawerContent: FunctionComponent<ConnectorDrawerContentProps> =
```
Comment (gashcrumb) [inline src/app/components/ConnectorDrawer/ConnectorDrawer.tsx:218]: I really like what you did in here, it's a lot more understandable :+1: .

File: src/app/components/ConnectorInfoTextList/ConnectorInfoTextList.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+    connectors_deployed: 1,
+  },
+};
+
+export default {
+  title: 'UI/Components/ConnectorInfoTextList',
+  component: ConnectorInfoTextList,
+};
+
+const Template = (args) => <ConnectorInfoTextList {...args} />;
```
Comment (gashcrumb) [inline src/app/components/ConnectorInfoTextList/ConnectorInfoTextList.stories.tsx:70]: This is awesome, thanks for the additional stories!

File: src/app/components/ConnectorStatus/ConnectorStatus.stories.tsx [hunk 6]
```diff
@@ -26,7 +26,7 @@ export enum ConnectorStatusValues {
 }
 
 export default {
-  title: 'UI/Connector/Statuses',
+  title: 'UI/Components/Statuses',
```
Comment (gashcrumb) [inline src/app/components/ConnectorStatus/ConnectorStatus.stories.tsx:29]: :+1:

**Review by gashcrumb — CHANGES_REQUESTED** [review]
Thanks for fixing storybook, that's great!  There's one question around `connectorData` that I commented on that's worth a second look.

**Review by uidoyen — APPROVED** [review]
Tested locally. Looks good :+1:

File: src/app/components/ConnectorInfoTextList/ConnectorInfoTextList.tsx (hunk 1 above)
Comment (indraraj) [reply src/app/components/ConnectorInfoTextList/ConnectorInfoTextList.tsx]: Done

File: src/app/components/ConnectorDrawer/ConnectorDrawer.tsx (hunk 2 above)
Comment (indraraj) [reply src/app/components/ConnectorDrawer/ConnectorDrawer.tsx]: I guess the original implementation was with "connector" and later the state machine was used to get the latest connector data from pooling to update the connector status accordingly.

**Comment by indraraj** [comment]
Good catch @gashcrumb, I have missed it in refactoring. Redundant source for the "connector" data was used to get the latest value from polling to update the connector state value as it changes. 

I have updated it to use only the state machine.

File: src/app/components/ConnectorDrawer/ConnectorDrawer.tsx (hunk 3 above)
Comment (indraraj) [reply src/app/components/ConnectorDrawer/ConnectorDrawer.tsx]: replied above.

**Review by gashcrumb — APPROVED** [review]
Great, thanks @indraraj this looks great!  Should we include this in today's rollout?

**Comment by indraraj** [comment]
I think we can skip this @gashcrumb, as it's mainly around refactoring than adding new items.

**Pull request merged**